Sources/ApodiniDatabase/Handlers/Delete.swift

```diff
@@ -0,0 +1,28 @@
+import Fluent
+import Apodini
+@_implementationOnly import Vapor
+
+/// A `Handler` that deletes the object with the given `IDValue` in the database, if it exists. If not an error is thrown.
+/// It uses the database that has been specified in the `DatabaseConfiguration`.
+public struct Delete<Model: DatabaseModel>: Handler {
+    @Apodini.Environment(\.database)
+    private var database: Fluent.Database
+
+    @Parameter(.http(.path))
+    var id: UUID
+    
+    var idParameter: Parameter<UUID> {
```

Comment:
As far as I can see the `idParameter` is used for testing purposes only. You could achieve the same result using reflection (`Mirror`) in the test case which would reduce the internal interface of the type.